### PR TITLE
Building test created!

### PR DESCRIPTION
**Pull request by mathlord2**
- Created test for making a building table and adding polygons to it

### CODE REVIEW

File: unweaver/databases/geopackage/feature_table.py [hunk 1]
```diff
@@ -629,25 +630,27 @@ def _row_value_generator(
...
             yield d.get(c, None)
 
+    # Put data into database (TODO: test this out for Polygons)
     def _serialize_geometry(self, geometry: BaseGeometry) -> str:
         # TODO: handle fewer types? Should standardize the inputs
-        if isinstance(geometry, LineString) or isinstance(geometry, Point):
+        if isinstance(geometry, LineString) or isinstance(geometry, Point) or isinstance(geometry, Polygon): # normal types
             return self._gp_header + geometry.wkb
-        elif isinstance(geometry, GeoJSONLineString) or isinstance(
+        elif isinstance(geometry, GeoJSONLineString) or isinstance( # GeoJSON types, not a dictionary yet
```
Comment (nbolten) [inline unweaver/databases/geopackage/feature_table.py:638]: Let's put this comment on a newline!

File: unweaver/databases/geopackage/feature_table.py [hunk 2]
```diff
@@ -629,25 +630,27 @@ def _row_value_generator(
...
             geometry, GeoJSONPoint
-        ):
+        ) or isinstance(geometry, GeoJSONPolygon):
             return self._gp_header + geomet.wkb.dumps(asdict(geometry))
-        elif isinstance(geometry, dict):
+        elif isinstance(geometry, dict): # Already a dictionary
```
Comment (nbolten) [inline unweaver/databases/geopackage/feature_table.py:642]: Let's put this comment on a newline!

File: unweaver/databases/geopackage/feature_table.py [hunk 3]
```diff
@@ -629,25 +630,27 @@ def _row_value_generator(
...
             print(type(geometry), geometry)
             raise ValueError("Invalid geometry")
 
+    # Gets data out the database (TODO: test this out for Polygons)
     def _deserialize_geometry(self, geometry: str) -> BaseGeometry:
         # TODO: use geomet's built-in GPKG support?
         header_len = len(self._gp_header)
         wkb = geometry[header_len:]
-        return geomet.wkb.loads(wkb)
+        return geomet.wkb.loads(wkb) # GeoJSON representation (dict)
```
Comment (nbolten) [inline unweaver/databases/geopackage/feature_table.py:653]: Let's put this comment on a newline!

File: unweaver/geojson.py [hunk 4]
```diff
@@ -17,17 +17,20 @@ class LineString:
     coordinates: List[Position]
     type: Literal["LineString"] = field(default="LineString", init=False)
 
+# TODO: define a Polygon dataclass
```
Comment (nbolten) [inline unweaver/geojson.py:20]: Remove this comment, since you made a Polygon dataclass!

**Review by nbolten — COMMENTED** [review]
Looking good! Just spruce up some comments (noted in the review) and it's good to go!

**Comment by mathlord2** [comment]
@nbolten I cleaned up the comments :)

**Pull request merged**